package.json

```diff
@@ -16,13 +16,15 @@
     "test": "nodemon -q -x 'yarn test:unit --silent'",
     "test:unit": "tape src/**/*.spec.js | tap-difflet",
     "test:feature": "cucumber-js test/features",
+    "test:components": "jest --silent=true \"src/app/.*\\.spec\\.js\"",
     "ci": "yarn && yarn lint && yarn test:unit && yarn test:feature"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "@emotion/react": "11.1.5",
     "@emotion/styled": "11.3.0",
+    "@mui/material": "^5.11.14",
```

Comment:
I wasn't able to get this running with the Docker image used in this repo

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/23480260/227982773-c1ef8b9e-60b0-4244-a08b-6900be9eb228.png">

Reply:
I am able to build it on my machine by running `make dev`

Can it be a docker's cache issue? 🤔  maybe running `docker builder prune` or `docker system prune` will help.
reference:
https://github.com/gregnb/mui-datatables/issues/1845
https://forums.docker.com/t/how-to-delete-cache/5753/2